nucleus/common/common-util/src/main/java/com/sun/enterprise/util/FelixPrettyPrinter.java

```diff
@@ -158,4 +161,9 @@ private static void printLn(StringBuilder messageBuilder, int indent, String mes
         }
         messageBuilder.append(message.trim()).append("\n");
     }
+
+    public static void main(String[] args) {
```

Comment:
I feel that this rather belongs to `FelixPrettyPrinterTest`.

Reply:
I agree, kill main :-)

Reply:
I use main to quickly copy and paste a stack into the pretty printer. It's not about testing really, but about convenience when you get a result string in the logs that for some reason has not been pretty printed.